discovery/gossiper.go

```diff
@@ -323,6 +323,112 @@ func (d *AuthenticatedGossiper) ProcessLocalAnnouncement(msg lnwire.Message,
 	return nMsg.err
 }
 
+// ChannelUpdateID is a unique identifier for ChannelUpdate messages, as
```

Comment:
Minor nit: this doesn't need to be publicly exported, as it's only used for dedup purposes within the gossiper.

The same can be said for the `Flags` field within the struct as well.